### DISCUSSION TITLE
Calendar auto slash needs added on input.

### DISCUSSION THREAD

**Discussion by JaylenBoykinSH** [primefaces/community#377] [PrimeNG] [answered]
A feature request that would be helpful is auto adding the slashes'/' when inputting the date using p calendar

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for the idea and feedback, in our internal backlog we plan to implement inputMask on the calendar to allow users to customize their input. But the priority right now is the accessibility implementation, we'll add this into our internal roadmap.

**Comment by lixaotec** [comment]
+1 to this, I also struggle with it

**Comment by cetincakiroglu** [comment, marked as answer]
You can use slotChar btw. https://primeng.org/installation
